### DISCUSSION TITLE
Should presenting the root cert count as a critical issue?

### DISCUSSION THREAD

**Discussion by roddie-digital** [matteocorti/check_ssl_cert#524] [General]
Hello - first off, thanks for this amazing script, I've had a few occasions now where it's been able to highlight issues where everything else had failed to. I wasn't sure to raise this as an issue so posting it here first. I noticed a when using the `--check-chain` option that if a host is presenting the root certificate the script will generate a `CRITICAL` state rather than `WARNING`. I wanted to check if this is the intended behaviour because starting on line [6231](https://github.com/matteocorti/check_ssl_cert/blob/master/check_ssl_cert#L6231) there is a comment saying it should raise a warning.

Thanks again

**Comment by matteocorti** [comment]
Good point. It was a warning and then after discussing it with someone, I changed it to a critical. I don't remember why, but I'll try to search for the reason (in a commit / change / issue).

**Comment by matteocorti** [comment]
```git log -L 6231,6256:check_ssl_cert``` does not really help: it was never a warning.
